### DISCUSSION TITLE
Broken accents/diacritics

### DISCUSSION THREAD

**Discussion by ghost** [tauri-apps/tauri#3136] [Q&A]
When typing accents/diacritics inside an input, it works fine:
![Kooha-12-29-2021-22-41-25](https://user-images.githubusercontent.com/23178554/147717445-a768067d-1bcf-41a3-a270-80dce8d5bcd7.gif)

But typing inside this _contenteditable_ element, it puts the accent and then the character with the same accent:
![Kooha-12-29-2021-22-42-02](https://user-images.githubusercontent.com/23178554/147717455-fdc587b1-2b80-4f4a-a547-8f94288a8223.gif)

All of them (that works or doesn't), log this in the terminal (this is a `pnpm exec tauri dev` with vite task):
```bash
22:37:20 [vite] page reload ts/home.ts (x2)
Couldn't get key from code: BracketLeft
Couldn't get key from code: BracketLeft
Couldn't get key from code: Quote
Couldn't get key from code: BracketLeft
Couldn't get key from code: Quote
```

I didn't open a bug because I can't reproduce in a isolated context. I tried creating an `<div>` with _contenteditable_ attribute, but it works as expected, same thing for `<textarea>` for example. In the one that doesn't work, I'm using [Milkdown](milkdown.dev/).

If anyone could help me trace down a reproducible bug I would appreciate it. Or if could help some dev/maintainer trace it down, also works.

-----

Operating System - Manjaro on GNOME (Wayland), version 21.2.0 X64

Node.js environment
  Node.js - 17.1.0
  @tauri-apps/cli - 1.0.0-beta.10

**Comment by probablykasper** [comment]
In general, anything happening in the html/css/js should be unrelated to Tauri. Does this happen if you run it in a browser?

**Reply by ghost** [reply]
Nope, it doesn't, neither in Firefox nor Chromium-based browsers.

But I agree with you, this may be unrelated to Tauri, I'm not sure.

Isn't "Couldn't get key from code: Quote" coming from Tauri logs?

**Reply by probablykasper** [reply]
Should be from tauri, yes (specifically tao). Might be something unsupported, I'd recommend opening an issue
